protx/utils/demographics.py

```diff
@@ -376,12 +376,12 @@ def get_age_race_pie_charts(area, geoid):
     from demographics d
     left join display_data u on
         u.NAME = d.DEMOGRAPHICS_NAME
-    where d.GEOTYPE = "{area}" and
-        d.UNITS = "{units}" and
-        d.DEMOGRAPHICS_NAME in ({variables}) and
-        d.GEOID = "{geoid}" and
-        d.YEAR = "2020" and
-        d.GEOTYPE = "{area}";
+    where (d.GEOTYPE = "{area}") and
```

Comment:
- Is there any preference for using `and` over `&&` in the code?
- If this is better for staying aligned with _code from the data team side_ then that makes sense.
- I think `&&` is more explicit/less ambiguous to read in multiline code. One could easily misinterpret the `and` as plain strings instead of an operand when written in multiline form.  
- But it works both ways so approving regardless! =)

Reply:
I think this mistakenly happens from the code switch between python and sql here but this is how the query was to start. Do you want me to change all the queries in this file to be more sql like?

Reply:
Nah, now that I grok the sql --> python munging I don't think it matters so much. I would do whatever best aligns with future changes by the data group (ie - if they update the plotly code later).